values.yaml

```diff
@@ -61,6 +61,14 @@ externalDB:
   database: ''
   host: ''
 
+##
+## To use external solr enable solr, provide host and port
+##
+solr:
+  enabled: false
+  host: ""
```

Comment:
Let's not keep it an empty string. 
Add default host and port.

Reply:
Yeah, this way it'll look cleaner, and give users more clarity about the data types.